README.md

```diff
@@ -13,6 +13,41 @@ To check WPCS and lint errors via CLI, run the following from the root directory
 To automatically fix errors and beautify files, run the following from the root directory.
 `./vendor/bin/phpcbf [filename]`
 
+### Activating Calypsoify
+
+To Calypsoify the dashboard and test various functionality in this plugin, there are a number of conditions that must be met.
+
+#### Plugin Dependencies
+* WooCommerce >= 3.0.0
```

Comment:
Should we mention Storefront, Powerpack, and the other bundled Storefront extensions (Blog Customizer, Parallax, Hero, Product Hero, Reviews)?

They aren't really required for most of the code in wc-calypso-bridge, but are needed for fully testing the customizer and experience.

Reply:
I guess if we do that we should also list out the other bundled plugins?

Reply:
@justinshreve Good idea.  Just pushed [this](https://github.com/Automattic/wc-calypso-bridge/pull/412/commits/7f5989fa49d2289d883c4653d8085dfc3a11d367) - does that look like a comprehensive list to you?